src/router/routes.tsx

```diff
@@ -0,0 +1,24 @@
+import { RouteObject } from 'react-router-dom';
```

Comment:
This should become 

```suggestion
import type { RouteObject } from 'react-router-dom';
```

or 
```suggestion
import { type RouteObject } from 'react-router-dom';
```